api/rpc/stat/stat.proto

```diff
@@ -3,14 +3,29 @@ syntax = "proto3";
 package stat;
 
 service Stat {
-  rpc ExecuteQuery (QueryRequest) returns (QueryReply) {}
+  rpc CPUQuery (CPURequest) returns (CPUReply) {}
 }
 
-message QueryRequest {
-  string database = 1;
-  string query=2;
+message CPURequest {
+  string ressourceName=1;
+  string filterDatacenter=2;
+  string filterServiceName=3;
+  string filterServiceID=4;
+  string filterContainerName=5;
+  string filterContainerID=6;
+  string filterContainerImage=7;
+  string filterNodeID=8;
 }
 
-message QueryReply {
-  string response = 1;
+message CPUEntry {
```

Comment:
in fact it's the number of CPU ticks per type (user, system, kernel, total) so integer, then the % can be computed dividing user / total or system / total in float and it can be computed the same way even with group by statement for instance because int can be added considering the / is done at the end.